src/CoreWCF.Http/tests/ClientContract/IXmlSerializerContract.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace ClientContract
+{
+	[AttributeUsage(AttributeTargets.All, Inherited = false)]
+	internal sealed class __DynamicallyInvokableAttribute : Attribute
+	{
+	}
+
+	public interface IXmlSerializable
```

Comment:
I don't quite understand what's going on here. This is just a reimplementation of the IXmlSerializable interface and the serializer won't use this interface as it's not the same type as System.Xml.Serialization.IXmlSerializable. Did the original test include this duplicated?